Lib/test/test_dis.py

```diff
@@ -2432,25 +2432,25 @@ def _unroll_caches_as_Instructions(instrs, show_caches=False):
 
 class TestDisCLI(unittest.TestCase):
 
-    def infile(self, content):
-        filename = tempfile.mktemp()
-        self.addCleanup(os_helper.unlink, filename)
```

Comment:
Ah sorry, I missed that you had the cleanup before (I was looking for a tearDown).

Reply:
Oh. Yes, I prefer the cleanup (saves me a few lines + a tearDown() method).